centreon/packages/ui/src/RichTextEditor/plugins/ToolbarPlugin/ListButton.tsx

```diff
@@ -0,0 +1,152 @@
+    label: 'Ordered List',
+    value: 'number'
+  }
+];
+
+interface Props {
+  disabled?: boolean;
+}
+
+const ListButton = ({ disabled }: Props): JSX.Element => {
+  const { classes } = useStyles();
+
+  const [editor] = useLexicalComposerContext();
+
+  const [elementList, setElementList] = useState(null);
+
+  const formatBulletList = (): void => {
+    if (elementList !== 'bullet') {
+      editor.dispatchCommand(INSERT_UNORDERED_LIST_COMMAND, undefined);
+    } else {
+      editor.dispatchCommand(REMOVE_LIST_COMMAND, undefined);
+    }
+  };
+
+  const formatNumberedList = (): void => {
+    if (elementList !== 'number') {
+      editor.dispatchCommand(INSERT_ORDERED_LIST_COMMAND, undefined);
+    } else {
+      editor.dispatchCommand(REMOVE_LIST_COMMAND, undefined);
+    }
+  };
+
+  const formatList = (type): void => {
+    if (equals(type, elementList)) {
+      setElementList(null);
+    }
+
+    if (equals(type, 'bullet')) {
+      formatBulletList();
+
+      return;
+    }
+    formatNumberedList();
+  };
+
+  const updateToolbar = useCallback(() => {
+    const selection = $getSelection();
+    const anchorNode = selection?.anchor.getNode();
+    const element = equals(anchorNode?.getKey(), 'root')
+      ? anchorNode
+      : $findMatchingParent(anchorNode, (e) => {
+          const parent = e.getParent();
+
+          return parent !== null && $isRootOrShadowRoot(parent);
+        }) || anchorNode?.getTopLevelElementOrThrow();
+
+    const elementKey = element?.getKey();
+    const elementDOM = editor.getElementByKey(elementKey);
+
+    if (isNil(elementDOM)) {
+      return;
+    }
+
+    if ($isListNode(element)) {
+      const parentList = $getNearestNodeOfType(anchorNode, ListNode);
+      const type = parentList
+        ? parentList.getListType()
+        : element.getListType();
+      setElementList(type);
+    }
+  }, [editor]);
+
+  const selectedList = options.find(({ value }) => equals(value, elementList));
+
+  useEffect(() => {
+    return editor.registerUpdateListener(({ editorState }) => {
+      editorState.read(() => {
+        updateToolbar();
+      });
+    });
+  }, [editor, updateToolbar]);
+
+  useEffect(() => {
+    return mergeRegister(
+      editor.registerUpdateListener(({ editorState }) => {
+        editorState.read(() => {
+          updateToolbar();
+        });
+      })
+    );
+  }, [editor, updateToolbar]);
+
+  return (
+    <Menu>
+      <Menu.Button
+        ariaLabel="list"
+        className={classes.button}
+        disabled={disabled}
+      >
+        {selectedList ? <selectedList.Icon /> : <UnorderedListIcon />}
+      </Menu.Button>
+      <Menu.Items className={classes.menuItems}>
+        <div className={classes.menu}>
+          {options.map(({ Icon, value, label }) => (
+            <Menu.Item
+              isActive={equals(value, elementList)}
+              key={value}
+              onClick={() => formatList(value)}
+            >
+              <Icon aria-label={label} />
```

Comment:
Translate aria labels

Reply:
is it necessary to add a translation to the area label?